### PR TITLE
RF: avoid 0 count reports in commit messages

### PR DESCRIPTION
**Pull request by yarikoptic**
untested... should we add a CI run? ;-)

### CODE REVIEW

File: find-datalad-repos.py [hunk 1]
```diff
@@ -283,9 +283,13 @@ def get_new_collection(self, searcher: "GHDataladSearcher") -> List[DataladRepo]
         return collection
 
     def get_report(self) -> str:
+        news = (
+            "{self.new_repos} new datasets",
+            "{self.new_runs} new `datalad run` users",
```
Comment (jwodder) [inline find-datalad-repos.py:288]: You forgot to make these f-strings.

File: find-datalad-repos.py [hunk 2]
```diff
@@ -283,9 +283,13 @@ def get_new_collection(self, searcher: "GHDataladSearcher") -> List[DataladRepo]
...
+        )
         return (
-            f"Added {self.new_hits} new hits: {self.new_repos} new datasets"
-            f" and {self.new_runs} new `datalad run` users"
+            f"Added {self.new_hits} new hits: " +
```
Comment (jwodder) [inline find-datalad-repos.py:291]: Are you sure you want to end with a ": " if there are no hits at all?  (There could still be things to commit if, say, a repo gained or lost stars.)

**Review by jwodder — CHANGES_REQUESTED** [review]

File: find-datalad-repos.py (hunk 2 above)
Comment (yarikoptic) [reply find-datalad-repos.py:291]: Good point! Will tune it up. Thank you!

Comment (yarikoptic) [reply find-datalad-repos.py]: done

File: find-datalad-repos.py (hunk 1 above)
Comment (yarikoptic) [reply find-datalad-repos.py]: done

**Review by jwodder — APPROVED** [review]

**Pull request merged**